modules/rhpam-apb/roles/deploy-amq/tasks/services.yml

```diff
@@ -87,6 +131,28 @@
           protocol: TCP
           targetPort: 61613
 
+- name: "[AMQ] set amq-stomp-ssl service state to {{ state }}"
+  k8s:
+    state: '{{ state }}'
+    definition:
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: '{{ application_name_amq }}-stomp-ssl'
+        namespace: '{{ namespace }}'
+        labels:
+          app: '{{ kieserver_deployment_name }}'
+          application: '{{ application_name }}'
+          service: '{{ amq_service_name }}'
+      spec:
+        selector:
+          deploymentConfig: '{{ amq_service_name }}'
+        ports:
+          - name: amq-stomp-ssl
```

Comment:
why not use the same name as in the DC? `stomp` or viceversa?

Reply:
not sure if I understand. Could you please clarify?
If you are talking about the port name I don't think it is necessary, this is the name that will be shown under services information on amq pod.

Reply:
It's just for consistency. It's fine if you prefer to keep using this name